content/blog/2017/2017-02-14-pipleine-editor-preview.adoc

```diff
@@ -25,19 +25,13 @@ A pipeline you create or edit visually can produce a link://blog/2017/02/03/decl
 
 image::/images/blueocean/editor-1.png["The Editor", role="center"]
 
-=== Whats next?
+=== What are we doing next?
 
-`Jenkinsfile`s belong in the Git repository so that it is versioned a long side
-your application code. In Blue Ocean, you will be able to edit a `Jenkinsfile`
-for a branch directly from within the user interface using the Visual Pipeline Editor.
+Jenkinsfiles belong in the Git repository so that it is versioned a long side
```

Comment:
"Jenkinsfiles belong in the Git repository so that it is versioned" still broken grammar